### PR TITLE
Create a Ukrainian-language package.

### PR DESCRIPTION
**Pull request by Peluhnya**
Create a Ukrainian-language package.
Translation from Russian is done. 
We plan to develop and improve the translation.
Tested on VScode - 1.88.0-insider

### CODE REVIEW

**Comment by Peluhnya** [comment]
@microsoft-github-policy-service agree

**Comment by CrazyTapok-bit** [comment]
This is great. But will they agree...

**Comment by Peluhnya** [comment]
> This is great. But will they agree...

We'll see what happens.
But it would be great if someone else would join us.

File: i18n/vscode-language-pack-uk/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Мовний пакет для української мови для VS Code
+
+Мовний пакет для української мови містить локалізацію інтерфейсу VS Code.
+
+## Использование
```
Comment (oleks) [inline i18n/vscode-language-pack-uk/README.md:5]: Використання

**Comment by oleks** [comment]
How do you solve [this issue](https://github.com/microsoft/vscode-loc/issues/1424#issuecomment-1634675274)?

File: i18n/vscode-language-pack-uk/README.md (hunk 1 above)
Comment (Peluhnya) [reply i18n/vscode-language-pack-uk/README.md:5]: Thank you)!

**Comment by koliasa** [comment]
Якраз шукав за пакетом в маркеті найшов половину, завтра спробую встановити цей, дуже файно що зявляється Українська мова

**Comment by Peluhnya** [comment]
> How do you solve [this issue](https://github.com/microsoft/vscode-loc/issues/1424#issuecomment-1634675274)?

OU.
I did not know about this problem. Next Sunday I'll try it with export, see what's what and then write back.

Thank you very much!

**Comment by okineadev** [comment]
WTF!

How did you do it?
I also somehow almost completely did the localization manually from JSON, but it was not worth doing, so I tried to do it according to the [official documentation](https://github.com/Microsoft/vscode-loc/wiki/How-to-create-language-pack-extension), but I could not import the translated xlf strings :(

**Comment by demiusdelph-developer** [comment]
Hello, this is a great project and a long-awaited one, but I don't understand why there is no ready-made solution in the marketplace in six months? I downloaded the vsix file from the localization files, installed it, everything works fine, but I don't understand if I should write if I find errors or suggestions, because everything stopped a long time ago and remains so

**Comment by okineadev** [comment]
> Hello, this is a great project and a long-awaited one, but I don't understand why there is no ready-made solution in the marketplace in six months? I downloaded the vsix file from the localization files, installed it, everything works fine, but I don't understand if I should write if I find errors or suggestions, because everything stopped a long time ago and remains so

Not at the moment, because we recently decided to do everything according to the official instructions and completely redo the translation, but I am very busy right now, so maybe in two weeks we will resume work on the translation. 80% of the way has already been completed, we literally only have to translate the lines with machine translation and translate the translations into JSON and that's it

**Comment by demiusdelph-developer** [comment]
> Not at the moment, because we recently decided to do everything according to the official instructions and completely redo the translation, but I am very busy right now, so maybe in two weeks we will resume work on the translation. 80% of the way has already been completed, we literally only have to translate the lines with machine translation and translate the translations into JSON and that's it

Understood, thanks for the answer and your work, the translation is already quite usable, but I will keep an eye on the topic

**Comment by METROKOP228** [comment]
Can I help you with something? I would like to participate in this project, do you need some help with translation?

**Comment by okineadev** [comment]
> Can I help you with something? I would like to participate in this project, do you need some help with translation?

Of course! Write to me in Telegram - https://t.me/okinea

**Review by anfrantic — APPROVED** [review]

**Comment by okineadev** [comment]
Watta sigma

**Pull request closed without merging**